java/src/Ice/src/main/java/com/zeroc/Ice/LoggerMiddleware.java

```diff
@@ -70,16 +57,16 @@ public CompletionStage<OutgoingResponse> dispatch(IncomingRequest request)
                 warning(ex.toString(), request.current);
             }
             throw ex;
-        } catch (RuntimeException | java.lang.Error ex) {
+        } catch (RuntimeException | Error ex) {
             warning(ex.toString(), request.current);
             throw ex;
         }
     }
 
     private void warning(String exceptionDetails, Current current) {
-        java.io.StringWriter sw = new java.io.StringWriter();
-        java.io.PrintWriter pw = new java.io.PrintWriter(sw);
-        com.zeroc.IceUtilInternal.OutputBase out = new com.zeroc.IceUtilInternal.OutputBase(pw);
+        var sw = new java.io.StringWriter();
+        var pw = new java.io.PrintWriter(sw);
+        var out = new OutputBase(pw);
```

Comment:
I think we should remove `OutputBase` and `ValueWriter` in a follow up PR.

Reply:
`ValueWriter` was removed in this PR.
It was completely unused.

I'll open an issue to remove `OutputBase` in the future.